pyccl/halos/halo_model.py

```diff
@@ -988,6 +988,163 @@ def halomod_Tk3D_1h(cosmo, hmc,
     return tk3d
 
 
+def halomod_Tk3D_SSC_linear_bias(cosmo, hmc, prof, bias1=1, bias2=1, bias3=1,
+                                 bias4=1,
+                                 is_number_counts1=False,
+                                 is_number_counts2=False,
+                                 is_number_counts3=False,
+                                 is_number_counts4=False,
+                                 p_of_k_a=None, lk_arr=None,
+                                 a_arr=None, extrap_order_lok=1,
+                                 extrap_order_hik=1, use_log=False):
+    """ Returns a :class:`~pyccl.tk3d.Tk3D` object containing
+    the super-sample covariance trispectrum, given by the tensor
+    product of the power spectrum responses associated with the
+    two pairs of quantities being correlated. Each response is
+    calculated as:
+
+    .. math::
+        \\frac{\\partial P_{u,v}(k)}{\\partial\\delta_L} =
+        \\left(\\frac{68}{21}-\\frac{d\\log k^3P_L(k)}{d\\log k}\\right)
+        b_u b_v P_L(k)+I^1_2(k|u,v) - (b_{u} + b_{v})
+        P_{u,v}(k)
+
+    where the :math:`I^1_2` is defined in the documentation
+    :meth:`~HMCalculator.I_1_2` and :math:`b_{}` and :math:`b_{vv}` are the
+    linear halo biases for quantities :math:`u` and :math:`v`, respectively
+    (zero if they are not clustering).
+
+    Args:
+        cosmo (:class:`~pyccl.core.Cosmology`): a Cosmology object.
+        hmc (:class:`HMCalculator`): a halo model calculator.
+        prof (:class:`~pyccl.halos.profiles.HaloProfile`): halo NFW
+            profile.
+        bias1 (float or array): linear galaxy bias for quantity 1. If an array,
+        it has to have the shape (a_arr, lk_arr).
```

Comment:
should it be `(len(a_arr), len(lk_arr))`?
Also, I think it'd be best if these were just of shape `(len(a_arr))` or floats

Reply:
Done.